Makefile

```diff
@@ -599,12 +599,14 @@ cover_ci: $(COVER_ROOT)/cover.out $(BIN)/goveralls
 	$(BIN)/goveralls -coverprofile=$(COVER_ROOT)/cover.out -service=buildkite || echo Coveralls failed;
 
 install-schema: cadence-cassandra-tool
+	$Q echo installing schema
```

Comment:
not complaining or anything, nor requesting it be removed, it just seems a bit unnecessary - `./cadence-cassandra-tool -k cadence setup-schema -v 0.0` will be printed a second later

Reply:
good point :) I'll clean it up in a following PR. I was just confused when was debugging failures.